src/evaluation_system/tests/plugin_manager_test.py

```diff
@@ -319,6 +319,45 @@ def test_get_history(dummy_settings, temp_user):
     assert g1[0] == g2[0]
 
 
+def test_get_history_all(dummy_settings, temp_user):
```

Comment:
it would be better to concatenate `test_get_history_all` with `test_get_history` and bring it afterward instead of defining a new test. just a minor suggestion

Reply:
I kept this test to keep the other test shorter, but yes, maybe is unnecessary. Will merge both.